<launch>

	<node pkg="final" type="PlannerNode.py" name="PlannerNode" output="screen">
        <param name="static_map" type="string" value="static_map" />
        <param name="halton_points" type="int" value="1250" />
        <param name="disc_radius" type="int" value="3" />
        <param name="collision_delta" type="double" value="0.15" />
        <param name="pub_topic" type="string" value="planner_node/car_plan" />
	</node>

</launch>
